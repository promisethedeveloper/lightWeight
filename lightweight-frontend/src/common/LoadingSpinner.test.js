import React from "react";
import { render } from "@testing-library/react";
import LoadingSpinner from "./LoadingSpinner";

/** Smoke test */
it("Should render without crashing", () => {
	render(<LoadingSpinner />);
});

/** Snapshot test */
it("should match snapshots", () => {
	const { asFragment } = render(<LoadingSpinner />);
	expect(asFragment()).toMatchSnapshot();
});

it("should show a 'Loading...' message", () => {
	const { queryByText } = render(<LoadingSpinner />);
	expect(queryByText("Loading...")).toBeInTheDocument();
});
