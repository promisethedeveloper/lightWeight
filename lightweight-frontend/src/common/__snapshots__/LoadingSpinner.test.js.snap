// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshots 1`] = `
<DocumentFragment>
  <div
    class="LoadingSpinner"
  >
    Loading...
  </div>
</DocumentFragment>
`;
